### PR TITLE
Flasher - fix for select2 height mismatch + padding

### PR DESCRIPTION
**Pull request by benlumley**
See discussion on #2837 

Addresses:

1. Mismatched height between select2 + regular select
2. The closeness of select boxes to the explanation to the side ?

I believe this is a better/cleaner fix than the platform specific CSS injection in #2837 

Screenshots:

ubuntu

ubuntu

mac 

mac

win

win

### CODE REVIEW

**Comment by Asizon** [comment]
Thanks looks good, i will test soon in my machine.

**Review by McGiverGim — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> PASS

**Review by Asizon — APPROVED** [review]

**Pull request merged**
